src/main/java/uk/gov/pay/directdebit/payments/dao/PaymentViewDao.java

```diff
@@ -27,6 +27,14 @@
             ":searchExtraFields " +
             "ORDER BY t.id DESC OFFSET :offset LIMIT :limit";
 
+    private final String COUNT_QUERY_STRING = "SELECT count(*) " +
+            "FROM transactions t " +
+            "INNER JOIN mandates m ON t.mandate_id = m.id " +
+            "INNER JOIN gateway_accounts ga ON ga.id = m.gateway_account_id " +
+            "LEFT JOIN payers pa ON t.mandate_id = pa.mandate_id " +
```

Comment:
out of interest do you know why this is a left join? Can there be a payment without a payer?

Reply:
In theory there shouldn't be a payment without payer. I just stripped down the search sql and to be honest I didn't pay attention to the left join. I shall remove it.